bin/fetch_ha_env.sh

```diff
@@ -21,9 +21,10 @@ require 'dotenv'
 Dotenv.load
 
 HA_LABEL = ARGV[0]
+ACCESS_TOKEN = ARGV[1] # Optional parameter used by CI
 
 def fetch_env
-  token = ENV.fetch("ACCESS_TOKEN")
+  token = ACCESS_TOKEN.nil? ? ENV.fetch("ACCESS_TOKEN") : ACCESS_TOKEN
```

Comment:
```suggestion
  token = ACCESS_TOKEN || ENV.fetch("ACCESS_TOKEN")
```
if access token has a value is `truthy` so it will short circuit, totally not important just wanted to share the option 😄 

You could also do:

```
ACCESS_TOKEN ||= ENV.fetch("ACCESS_TOKEN")
```

And then use `ACCESS_TOKEN` from that point forward 🤷

Reply:
Excellent!
I was looking for something like that, but my knowledge in Ruby is 0